.circleci/continue_config.yml

```diff
@@ -0,0 +1,779 @@
+            - 1.23.3
+        name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
+        requires:
+        - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+    - test:
+        matrix:
+          parameters:
+            arch:
+            - amd64
+            - arm64
+            waf:
+            - 'ON'
+            - 'OFF'
+            base-image:
+            - nginx:1.23.2-alpine
+            - nginx:1.23.2
+            nginx-version:
+            - 1.23.2
+        name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
+        requires:
+        - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+    - test:
+        matrix:
+          parameters:
+            arch:
+            - amd64
+            - arm64
+            waf:
+            - 'ON'
+            - 'OFF'
+            base-image:
+            - nginx:1.23.1-alpine
+            - nginx:1.23.1
+            nginx-version:
+            - 1.23.1
+        name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
+        requires:
+        - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+    - test:
+        matrix:
+          parameters:
+            arch:
+            - amd64
+            - arm64
+            waf:
+            - 'ON'
+            - 'OFF'
+            base-image:
+            - nginx:1.23.0-alpine
+            - nginx:1.23.0
+            nginx-version:
+            - 1.23.0
+        name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
+        requires:
+        - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+    - test:
+        matrix:
+          parameters:
+            arch:
+            - amd64
+            - arm64
+            waf:
+            - 'ON'
+            - 'OFF'
+            base-image:
+            - nginx:1.22.0-alpine
+            - nginx:1.22.0
+            nginx-version:
+            - 1.22.0
+        name: test << matrix.nginx-version >> on << matrix.base-image >>:<< matrix.arch >> WAF << matrix.waf >>
+        requires:
+        - build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
+  build-and-test-installer:
+    when:
+      and:
+        - not: << pipeline.git.tag >>
+        - << pipeline.parameters.run-installer-workflow >>
+    jobs:
+     - build_installer_amd64:
+        name: build installer on amd64
+     - build_installer_arm64:
+        name: build installer on arm64
+     - installer_test:
+        matrix:
+          parameters:
+            base-dockerfile:
+            - 'bookworm.Dockerfile'
+            - 'alpine.Dockerfile'
+            - 'al2023.Dockerfile'
+        name: test installer with << matrix.base-dockerfile >> base image
+        requires:
+        - build installer on amd64
```

Comment:
build-and-test-all is generated thanks to `bin/generate_job.rb`. Usually one appends the output of this script to this file. To maintain this, would it be possible to move this workflow before build-and-test-all?